custom_addons/incident_management/models/x_incident_record.py

```diff
@@ -23,6 +24,7 @@ def create(self, vals_list):
         incident = super().create(vals_list)
         # Call the send_email method
         incident.action_send_email()
+        incident.create_activity('Ready to go Investigation', 'To Do', incident.location.location_manager.id, self.due_date)
```

Comment:
Instead of 'Ready to go Investigation'
. Can you name it as 'Assign Investigation Team'